components/molecule/carousel/src/CarouselContainer.js

```diff
@@ -12,7 +12,9 @@ import slidy from './slidy.js'
 
 const CarouselContainer = ({
   arrowLeft,
+  arrowLeftLabel,
```

Comment:
We can define a default value for this two props.

```suggestion
  arrowLeftLabel = 'Previous',
  arrowRightLabel = 'Next'
```

Reply:
I've already defined the default value in the `index.js` @AgonisticKatai ,
https://github.com/SUI-Components/sui-components/pull/2877/files#diff-84c01abfbb6ae0dc76357fad5b9205fd28ecf832c663e369ad67639a7dc3fc71R22